src/css/Header.scss

```diff
@@ -17,13 +17,15 @@
 #app-header-desktop {
   grid-template-columns: minmax(100px, 500px) auto;
   grid-gap: 40px;
-  padding: 15px 30px;
+  padding: 0 30px;
```

Comment:
Why remove the padding here? Since you added extra margin on Searchpage, there is no difference, so it's probably best to leave it as is.

Reply:
It makes the header smaller so it doesn't take upp too much space of the screen